frontend/src/components/Login.jsx

```diff
@@ -0,0 +1,126 @@
+// Import React tools we need
+import { useState } from 'react'
+import './Login.css'
+
+// This component shows the login form where users sign in
+function Login({ onLogin, onSwitchToRegister }) {
+  // State to store what user types in the form
+  const [formData, setFormData] = useState({
+    email: '',    // User's email address
```

Comment:
you can remove the comments for this. It's obvious. You don't need to add comments everywhere. Sometimes what you're implementing is obvious in the code